scripts/Muon/GUI/Common/grouping_table_widget/grouping_table_widget_view.py

```diff
@@ -1,15 +1,15 @@
 from __future__ import (absolute_import, division, print_function)
 
-from PyQt4 import QtCore, QtGui
-from PyQt4.QtCore import pyqtSignal as Signal
+from qtpy import QtWidgets, QtGui, QtCore
```

Comment:
can we remove the QtCore import and just keep from qtpy.QtCore import Signal? I may have missed something but it appears that we only use Signal from QtCore in this file